app/models/setting/provisioning.rb

```diff
@@ -47,6 +47,7 @@ def self.default_settings
       self.set('manage_puppetca', N_("Foreman will automate certificate signing upon provision of new host"), true, N_('Manage PuppetCA')),
       self.set('ignore_puppet_facts_for_provisioning', N_("Stop updating IP address and MAC values from Puppet facts (affects all interfaces)"), false, N_('Ignore Puppet facts for provisioning')),
       self.set('ignored_interface_identifiers', N_("Ignore interfaces that match these values during facts importing, you can use * wildcard to match names with indexes e.g. macvtap*"), IGNORED_INTERFACES, N_('Ignore interfaces with matching identifier')),
+      self.set('ignored_fact_names', N_("Ignore fact names that match these values during facts importing, you can use * wildcard to match names with indexes e.g. macvtap*"), IGNORED_INTERFACES, N_('Ignore fact names with matching identifier')),
```

Comment:
Can you tell the user this includes puppet, rhsm and all facts in general?

Reply:
The default value and second part of description seems wrong :-)

Reply:
Well it is correct, but I agree that the constant should be renamed in this case.

Reply:
I didn't know there fact with names like macvtap*, perhaps fact names from newer facter. In such case, that's fine and just the constant should be named differently, it could be set to IGNORED_INTERFACES values though.